great_expectations/data_context/data_context/context_factory.py

```diff
@@ -326,7 +329,11 @@ def _get_cloud_context(  # noqa: PLR0913 # FIXME CoP
         cloud_organization_id: str | None = None,
         user_agent_str: str | None = None,
         cloud_mode: bool | None = None,
+        mode: ContextModes | None = None,
     ) -> CloudDataContext | None:
+        if cloud_mode is False:
```

Comment:
Completely non-blocking, just complaining. Why is this even a parameter? this is equivalent to calling `do_the_thing(actually_do_not_do_the_thing=True)`. I'm assuming it's because of upstream stuff that bled down to this layer?

Reply:
yeah, `cloud_mode` used to be the canonical way to get a CloudDataContext. And when we switched to the `mode` enum, we left it to avoid breaking changes. And we missed removing it in v1...

Reply:
it just means our API has two ways to do the same thing, like pretty much everywhere else 😄 bonus points for consistency?